vulnerability-report.md

```diff
@@ -0,0 +1,55 @@
+# A1 - Injection
+## How you found and exploited the vulnerability, including the code/method you used?
+- Uploading an image with sql in the file name allowed us to create an image that when clicked
+exposes the hashed username and password in the Django error message.  The exact sql is:
+    - testPic',(select password from auth_user where username='admin'),7);--
+
+## What you were able to do with the vulnerability (i.e. what was exposed?)
+- We were able to see the hashed password and username for the user with the error message, not that much else was revealed.
+
+## The code that fixes the vulnerability
+- Creating the file with defined values and saving the file would prevent sql injection, instead of directly executing the code to the database.
+
+
+# A2 - Broken Auth
+## How you found and exploited the vulnerability, including the code/method you used
+- On the user creation form we added checkboxes for is_admin, is_staff, is_superuser.
+    <input type="checkbox" value="is_admin" id="is_admin">
+    <input type="checkbox" value="is_staff" id="is_staff">
+    <input type="checkbox" value="is_superuser" id="is_superuser">
+
+## What you were able to do with the vulnerability (i.e. what was exposed?)
+- We were able to log into the admin site. Which is not good.
+
+## The code that fixes the vulnerability
+- We restricted the fields of the form to only modifiable categories we would want the user to hsve access to:
+    fields = ['username', 'first_name', 'last_name', 'email', 'password']
+
+
+# A3 - XSS
+## How you found and exploited the vulnerability, including the code/method you used
+- We edited the username to have an alert popup where the username would be loaded by inserting a script as the username to run.  The code used is:
+<script>alert('This worked as a hack.')</script>
+
+
+## What you were able to do with the vulnerability (i.e. what was exposed?)
+- We were able to run a script that would run each time a username aspect was run.   This would be more malicious with a vicous script rather then an alert saying there was a hack.
+
+
+## The code that fixes the vulnerability
+- We removed the '|safe' in the templates, so the tags would be removed and we do not presume that the username is safe to display.  
```

Comment:
You also activated `SECURE_BROWSER_XSS_FILTER = True`. You should document that.